File: Python/Program/Jump.py
import BasicMove

class Jump(BasicMove):
    def __init__(self, name, duration, difficulty, height):
        super().__init__(name, duration, difficulty)
        self.__jumpHeight = height
    
    def setJumpHeight(self, height):
        self.__jumpHeight = height
    def getJumpHight(self):
        return self.__jumpHeight
    

File: CPP/Program/Jump.cpp
#ifndef JUMP_CPP
#define JUMP_CPP
#include "BasicMove.cpp"

#include <string>

using namespace std;

class Jump : public BasicMove{
    private:
        int jumpHeight; // in cm
    public:
        Jump(string name, int duration, string difficulty, int height) : BasicMove(name, duration, difficulty){
            this->jumpHeight = height;
        }

        void setJumpHeight(int height){
            this->jumpHeight = height;
        }
        int getJumpHeight(){
            return this->jumpHeight;
        }

        ~Jump(){}
};